src/SelfUpdateCommand.php

```diff
@@ -48,6 +68,7 @@ protected function configure()
             ->addOption('stable', NULL, InputOption::VALUE_NONE, 'Use stable releases (default)')
             ->addOption('preview', NULL, InputOption::VALUE_NONE, 'Preview unstable (e.g., alpha, beta, etc.) releases')
             ->addOption('compatible', NULL, InputOption::VALUE_NONE, 'Stay on current major version')
+            ->addOption('version_constraint', NULL, InputOption::VALUE_REQUIRED, 'Apply version constraint')
```

Comment:
Please make this an optional argument rather than an option (e.g. `self:update ^2` to roll back to v2)

Reply:
Changed